cmd/kubeadm/app/util/certs/util.go

```diff
@@ -43,6 +43,19 @@ func SetupCertificateAuthority(t *testing.T) (*x509.Certificate, crypto.Signer)
 	return caCert, caKey
 }
 
+// SetupIntermediateCertificateAuthority is a utility function for kubeadm testing that creates a
+// Intermediate CertificateAuthority cert/key pair
+func SetupIntermediateCertificateAuthority(t *testing.T, parentCert *x509.Certificate, parentKey crypto.Signer) (*x509.Certificate, crypto.Signer) {
+	caCert, caKey, err := pkiutil.NewIntermediateCertificateAuthority(parentCert, parentKey, &pkiutil.CertConfig{
+		Config: certutil.Config{CommonName: "kubernetes intermediate CA"},
+	})
+	if err != nil {
+		t.Fatalf("failure while generating intermediate CA certificate and key: %v", err)
```

Comment:
```suggestion
		t.Fatalf("Failure while generating intermediate CA certificate and key: %v", err)
```

test output messages should be capitalized unlike non-test errors